elementary/monitor/api/alerts.py

```diff
@@ -50,6 +52,7 @@ def _query_alert_type(self, run_operation_args: dict, alert_factory_func: Callab
                 try:
                     alerts.append(alert_factory_func(
                         elementary_database_and_schema=self.elementary_database_and_schema,
+                        timezone=timezone,
```

Comment:
This should be `timezone=self.config.timezone`.

Reply:
done.